code/core/src/main/java/com/adobe/marketing/mobile/services/ui/vnext/floatingbutton/FloatingButtonSettings.kt

```diff
@@ -0,0 +1,63 @@
+/*
+  Copyright 2023 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.services.ui.vnext.floatingbutton
+
+import java.io.InputStream
+
+/**
+ * Settings for the FloatingButton
+ * @param height height of the button in dp
+ * @param width width of the button in dp
+ * @param initialGraphic initial graphic to be displayed on the button
+ * @param cornerRadius corner radius of the button in dp
+ */
+class FloatingButtonSettings private constructor(
+    val height: Int,
+    val width: Int,
+    val initialGraphic: InputStream?,
```

Comment:
After reviewing the code, my preference is to accept Bitmap instead of InputStream.